batch_writer.go

```diff
@@ -2,11 +2,22 @@ package ghostferry
 
 import (
 	"fmt"
+	"os"
+	"time"
+
 	sql "github.com/Shopify/ghostferry/sqlwrapper"
 
 	"github.com/sirupsen/logrus"
 )
 
+var batchWriteRetryDelay = 5 * time.Millisecond
+
+func init() {
```

Comment:
Can you only define one `init` per package? We can leave this code here and deal with it later, likely when we make this retry delay configurable.

Reply:
>Can you only define one init per package?

You can define more than one `init()` and it's common to have multiple inits per package..

Reply:
Cool. TIL.